src/main/java/org/ow2/proactive/procci/model/occi/metamodel/MixinBuilder.java

```diff
@@ -10,29 +10,42 @@
 import java.util.Set;
 import java.util.stream.Collectors;
 
-import org.ow2.proactive.procci.model.exception.CloudAutomationException;
+import org.ow2.proactive.procci.model.exception.ClientException;
 import org.ow2.proactive.procci.model.exception.MissingAttributesException;
 import org.ow2.proactive.procci.model.exception.SyntaxException;
 import org.ow2.proactive.procci.model.occi.infrastructure.constants.InfrastructureKinds;
 import org.ow2.proactive.procci.model.occi.metamodel.rendering.AttributeRendering;
 import org.ow2.proactive.procci.model.occi.metamodel.rendering.MixinRendering;
+import lombok.AccessLevel;
+import lombok.Getter;
 
 
 /**
  * Created by mael on 22/09/16.
  */
 public class MixinBuilder {
 
+    private ProviderMixin providerMixin;
+
+    @Getter(value = AccessLevel.PROTECTED)
```

Comment:
only Getter method for these fields, so they are supposed be final, so that you can use @Data at class level, and change all these fields as final